org.eclipse.buildship.kotlin/build.gradle

```diff
@@ -0,0 +1,21 @@
+buildscript {
+    ext.kotlin_version = '1.0.4'
+    repositories.jcenter()
+    dependencies.classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+}
+
+apply plugin: eclipsebuild.BundlePlugin
+apply plugin: 'kotlin'
+
+dependencies {
+    compile 'org.gradle:gradle-core:3.3'
+    compile 'org.gradle:gradle-process-services:3.3'
```

Comment:
This just shows that defining this stuff in the annotation doesn't make sense. It should be provided by the ScriptTemplateProvider. Same as with the isApplicable check. We should not need to load the template at all at this point.